content/en/logs/faq/how-to-tail-logs-from-host-using-a-container-agent.md

```diff
@@ -0,0 +1,148 @@
+---
+title: How to tail logs from host using the Container Agent
+kind: faq
+further_reading:
+- link: "/docker/log"
+  tag: "Documentation"
+  text: "Logging with Docker"
+- link: "/agent/kubernetes/log/?tab=daemonset#pagetitle"
+  tag: "Documentation"
+  text: "Logging with Kubernetes"
+---
+
+<div class="alert alert-info">Datadog recommends using <b>STDOUT/STDERR</b> to collect container logs. This is the official documentation on collecting logs from <a href="/docker/log/?tab=containerinstallation#overview" target="_blank">Docker</a> and <a href="/agent/kubernetes/log/?tab=daemonset#pagetitle" target="_blank">Kubernetes</a>. To learn more about Docker logging, read the <a href="https://www.datadoghq.com/blog/docker-logging/" target="_blank">Docker Logging Best Practices</a> blog post.</div>
```

Comment:
We should add that using STDOUT/STDERR automatically add container metadata to the logs which collecting through a file don't.